### PR TITLE
Added missing ctia code in clj-momo

### PR DESCRIPTION
**Pull request by yogsototh**
Subsume #9

The added files will be really helpful to be create trapperkeeper service for riemann.

### CODE REVIEW

File: project.clj [hunk 1]
```diff
@@ -22,7 +22,17 @@
                  [cheshire "5.6.3"]
 
                  ;;http
-                 [clj-http "3.4.1"]]
+                 [clj-http "3.4.1"]
+
+                 ;; Metrics
+                 [metosin/compojure-api "1.1.8"]
```
Comment (gbuisson) [inline project.clj:28]: I think we shouldn't bundle compojure-api on clj-momo.

**Comment by msprunck** [comment]
A riemann reporter already exists in the [metrics-clojure](https://github.com/sjl/metrics-clojure) library. Did you try it ? 

Furthermore, puppetlabs provides a trapperkeeper service for metrics [trapperkeeper-metrics](https://github.com/puppetlabs/trapperkeeper-metrics) with a graphite reporter. Riemann provides a graphite server to ingest events.

**Review by gbuisson — CHANGES_REQUESTED** [review]

File: project.clj (hunk 1 above)
Comment (polygloton) [reply project.clj:28]: Its used in the metrics code.

Comment (polygloton) [reply project.clj:28]: Its only for one fn though.  Maybe we could pass in the fn.

**Comment by polygloton** [comment]
Let's track this in #9

**Comment by polygloton** [comment]
I don't know the answer about the metrics-clojure library.  The issue that this work is for asks to extract the existing code out of CTIA into clj-momo.  If the code is not desired, that should be fixed in another issue.

**Pull request closed without merging**